.php_cs

```diff
@@ -5,5 +5,5 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 
 return Symfony\CS\Config\Config::create()
     ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
-    ->fixers(array('short_array_syntax'))
+    ->fixers(['short_array_syntax'])
```

Comment:
No. PHP-CS-Fixer is still base on PHP >= 5.3.6
This should be long array syntax
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.9/composer.json#L17